pytest_mock.py

```diff
@@ -10,7 +10,7 @@
 version = '1.2'
 
 
-class MockFixture(object):
+class MockFixture:
```

Comment:
I'd prefer that, as I have no idea how an old-style class works, and I'm sure I'm not the only one :wink: